src/lib/jitter/info.txt

```diff
@@ -0,0 +1,19 @@
+# https://botan.randombit.net/handbook/dev_ref/configure.html#adding-a-new-module
+
+load_on never
```

Comment:
This should probably be `load_on dep`  - `load_on never` is mostly just used for blacklisting a known broken module